deployments/helm/nvidia-dra-driver-gpu/templates/kubeletplugin.yaml

```diff
@@ -49,6 +49,22 @@ spec:
       serviceAccountName: {{ include "nvidia-dra-driver-gpu.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.kubeletPlugin.podSecurityContext | nindent 8 }}
+      initContainers:
+      - name: init-container
+        image: {{ include "nvidia-dra-driver-gpu.fullimage" . }}
+        securityContext:
+          privileged: true
+        command: [sh, -c, /usr/bin/kubelet-plugin-prestart.sh]
```

Comment:
you shouldn't need the `sh -c`

Reply:
Right, oversight. Interesting that this works. Updating.

Reply:
updated